moler/config/loggers.py

```diff
@@ -13,7 +14,7 @@
 import sys
 
 logging_path = os.getcwd()  # Logging path that is used as a prefix for log file paths
```

Comment:
maybe we should change `logging_path` to `_logging_path` to indicate "you should not use that global attribute directly" - it is now "private". And also check if there are no usages like:
```
from moler.config import loggers
...
loggers.logging_path = ...
```